Samples/AppLifecycle/Activation/cs-winforms-unpackaged/CsWinFormsActivation/Program.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Windows.Forms;
+using Microsoft.Windows.ApplicationModel.DynamicDependency;
 
 namespace CsWinFormsActivation
 {
```

Comment:
Replace lines 13-14 with

```
static var majorMinorVersion = global::Microsoft.WindowsAppSDK.Release.MajorMinor;
static var versionTag = global::Microsoft.WindowsAppSDK.Release.VersionTag;
static var minVersion = new global::Microsoft.Windows.ApplicationModel.DynamicDependency.PackageVersion(Microsoft.WindowsAppSDK.Runtime.Version.UInt64);
```

same applies to all

Reply:
Just to confirm @DrusTheAxe, is replacing the types with var intentional in your recommendation?

Reply:
Yes. It could go either way for majorMinorVersion and versionTag but general rule is objects are better as var when the right side of the = is the type e.g.

```
Widget x = new Widget();
```

is better as

```
var x = new Widget();
```

since it's unambiguous to the reader what the type of x is.

For majorMinorVersion and versionTag they're constants so for consistency `var` seemed better, though one could make a case for `uint` and `string`.